composer.json

```diff
@@ -12,14 +16,16 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3.0",
```

Comment:
Given the introduction of function return vars and argument type declarations, this minimum version should probably be 7.4.0